src/docker-osx-dev

```diff
@@ -441,7 +441,7 @@ function configure_docker_machine {
   DOCKER_HOST_NAME="$DOCKER_MACHINE_NAME"
   # Support < 0.4.1 and >= 0.5.1
   DOCKER_HOST_USER=$(inspect_docker_machine "{{.Driver.SSHUser}}" || inspect_docker_machine "{{.Driver.Driver.SSHUser}}")
-  DOCKER_HOST_IP=$(inspect_docker_machine "{{.Driver.IPAddress}}" || inspect_docker_machine "{{.Driver.Driver.IPAddress}}")
+  DOCKER_HOST_IP=$(docker-machine ip)
```

Comment:
The [ip command](https://docs.docker.com/machine/reference/ip/) takes the machine name as an arg. Shouldn't you use `$DOCKER_MACHINE_NAME` as a parameter here?